src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Pack.targets

```diff
@@ -218,15 +217,43 @@ Copyright (c) .NET Foundation. All rights reserved.
       Condition="'$(IncludeSource)' == 'true'"
       Projects="$(MSBuildProjectFullPath)"
       Targets="SourceFilesProjectOutputGroup"
-      Properties="TargetFramework=%(_TargetFrameworks.Identity);
-                  BuildProjectReferences=false;">
+      Properties="TargetFramework=%(_TargetFrameworks.Identity);">
 
       <Output
           TaskParameter="TargetOutputs"
           ItemName="_SourceFiles" />
     </MSBuild>
   </Target>
   
+  <Target Name="_GetBuildOutputFilesWithTfm"
+          DependsOnTargets="BuiltProjectOutputGroup;DocumentationProjectOutputGroup;SatelliteDllsProjectOutputGroup;_AddPriFileToPackBuildOutput;"
+          Returns="@(_TargetPathsToAssembliesWithTfm)">
+    <ItemGroup>
+      <_TargetPathsToAssembliesWithTfm Include="@(SatelliteDllsProjectOutputGroupOutput)">
+        <TargetFramework>$(TargetFramework)</TargetFramework>
```

Comment:
@emgarten  as discussed offline, there is no need for type , as the ```TargetPath``` metadata automatically adds culture to the filename.